From 8fd4afaa26fd8088d847b4aa242e2791f9b1156c Mon Sep 17 00:00:00 2001
From: Thiago Macieira <thiago.macieira@intel.com>
Date: Thu, 14 Apr 2016 03:44:11 +0000
Subject: [PATCH] Force configure not to bail out on unknown cmdline options

---
 mkspecs/features/qt_configure.prf | 6 +-----
 1 file changed, 1 insertion(+), 5 deletions(-)

diff --git a/mkspecs/features/qt_configure.prf b/mkspecs/features/qt_configure.prf
index 6537d98c5d..b3661924a3 100644
--- a/mkspecs/features/qt_configure.prf
+++ b/mkspecs/features/qt_configure.prf
@@ -350,11 +350,7 @@ defineTest(qtConfParseCommandLine) {
         }
 
         isEmpty(type) {
-            qtConfAddError("Unknown command line option '$$c'.")
-            equals(config.input.continue, yes): \
-                next()
-            else: \
-                return()
+            next()
         }
 
         call = "qtConfCommandline_$${type}"
-- 
2.26.0

